src/IO/Exporter/PointwiseInterpolator.cpp

```diff
@@ -129,6 +149,23 @@ void interpolate_to_points(
   }  // omp parallel
 }
 
+template <size_t Dim>
+void _interpolate_to_point(
```

Comment:
This function doesn't seem to be used anywhere. Is it necessary?

Reply:
Oh it will be used by the interpolator class in the next PR. I can remove it for now